library/core/src/panicking.rs

```diff
@@ -109,6 +111,28 @@ fn panic_no_unwind() -> ! {
     unsafe { panic_impl(&pi) }
 }
 
+/// Aborts the process, but with a properly displayed panic message.
+#[cold]
+#[rustc_allocator_nounwind]
+pub(crate) fn panic_abort<'a>(message: Option<&'a fmt::Arguments<'a>>) -> ! {
```

Comment:
```suggestion
pub(crate) fn panic_abort<'a>(message: fmt::Arguments<'a>) -> ! {
```
(and change others accordingly)

`PanicInfo`'s optional message is only for `panic_any`; for something new like nounwind panic, there's no reason to use `Option` here. It should just follow the signature of `panic_fmt` below.